src/js/order.js

```diff
@@ -699,62 +701,95 @@ function fetchYear(year, request_scheduler, nocache_top_level) {
             '?opt=ab&digitalOrders=1' +
             '&unifiedOrders=1' +
             '&returnTo=' +
             '&orderFilter=year-%(year)s' +
-            '&startIndex=%(startOrderPos)s'],
+            '&startIndex=%(startOrderPos)s' +
+            '&language=fr_FR'],
         'www.amazon.fr': ['https://%(site)s/gp/css/order-history' +
             '?opt=ab&digitalOrders=1' +
             '&unifiedOrders=1' +
             '&returnTo=' +
             '&orderFilter=year-%(year)s' +
-            '&startIndex=%(startOrderPos)s'],
+            '&startIndex=%(startOrderPos)s' +
+            '&language=fr_FR'],
         'smile.amazon.com': ['https://%(site)s/gp/css/order-history' +
             '?opt=ab&digitalOrders=1' +
             '&unifiedOrders=1' +
             '&returnTo=' +
             '&orderFilter=year-%(year)s' +
-            '&startIndex=%(startOrderPos)s'],
+            '&startIndex=%(startOrderPos)s' +
+            '&language=en_US',
+
+            'https://%(site)s/gp/your-account/order-history/ref=oh_aui_menu_yo_new_digital' +
+            '?ie=UTF8' +
+            '&digitalOrders=1' +
+            '&orderFilter=year-%(year)s' +
+            '&unifiedOrders=0' +
+            '&startIndex=%(startOrderPos)s' +
+            '&language=en_US'],
         'www.amazon.com': [
             'https://%(site)s/gp/your-account/order-history/ref=oh_aui_menu_date' +
             '?ie=UTF8' +
             '&orderFilter=year-%(year)s' +
-            '&startIndex=%(startOrderPos)s',
+            '&startIndex=%(startOrderPos)s' +
+            '&language=en_US',
+
             'https://%(site)s/gp/your-account/order-history/ref=oh_aui_menu_yo_new_digital' +
             '?ie=UTF8' +
             '&digitalOrders=1' +
             '&orderFilter=year-%(year)s' +
             '&unifiedOrders=0' +
-            '&startIndex=%(startOrderPos)s'],
+            '&startIndex=%(startOrderPos)s' +
+            '&language=en_US'],
         'www.amazon.com.mx': [
             'https://%(site)s/gp/your-account/order-history/ref=oh_aui_menu_date' +
             '?ie=UTF8' +
             '&orderFilter=year-%(year)s' +
-            '&startIndex=%(startOrderPos)s',
+            '&startIndex=%(startOrderPos)s' +
+            '&language=es_US',
+
             'https://%(site)s/gp/your-account/order-history/ref=oh_aui_menu_yo_new_digital' +
             '?ie=UTF8' +
             '&digitalOrders=1' +
             '&orderFilter=year-%(year)s' +
             '&unifiedOrders=0' +
-            '&startIndex=%(startOrderPos)s'],
+            '&startIndex=%(startOrderPos)s' +
+            '&language=es_US'],
+        'other': ['https://%(site)s/gp/css/order-history' +
+            '?opt=ab&digitalOrders=1' +
+            '&unifiedOrders=1' +
+            '&returnTo=' +
+            '&orderFilter=year-%(year)s' +
+            '&startIndex=%(startOrderPos)s' +
+            '&language=en_GB'],
+    }
+    let templates = templates_by_site[util.getSite()];
```

Comment:
templates can be const like this:

const site = util.getSite();
const templates = site in templates_by_site ? templates_by_site[site] : templates_by_site['other'];